src/RandomForestBase.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Can you disable the rule only for the line (s) where it's needed?  With `eslint-disable-next-line no-unused-vars'